scripts/systemd/serial-keel.template.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Serial Keel Server
+
+[Service]
+User=[user-name-here]
+WorkingDirectory=/home/[user-name-here]/serial-keel/
+ExecStartPre=/home/[user-name-here]/serial-keel/scripts/systemd/check_and_upgrade_serial_keel.sh [branch-here]
+ExecStart=/home/[user-name-here]/.cargo/bin/serial-keel /home/[user-name-here]/config.ron
```

Comment:
So since this is executing as the user, I'm hoping the `env` is inherited.
Then `/home/$user/.cargo/bin` should be on the `PATH`.

In that case we _should_ be able to just `ExecStart=serial-keel ...`.

If that is not possible (we can't inherit the users' env) then I propose we append `/home/[user-name-here]/.cargo/bin` to the `PATH` manually.

I found this: https://wiki.archlinux.org/title/systemd/User
Which shows use of /.config/environment.d/